src/backend/Csrs.Api/Models/Account.cs

```diff
@@ -18,19 +21,19 @@ public class PortalAccount
         [SwaggerSchema(Format = "date")]
         public DateTime DateOfBirth { get; set; }
         [Required]
-        public string? Gender { get; set; }
+        public LookupValue? Gender { get; set; }
         public string? AddressStreet1 { get; set; }
         public string? AddressStreet2 { get; set; }
         public string? City { get; set; }
-        public string? Province { get; set; }
+        public LookupValue? Province { get; set; }
```

Comment:
bit undecided about changing this to LookupValue, really only need to have a `int? ProvinceId` value. Open for discussion.